### PR TITLE
add option inherit advance from alert group in the alert rule as the default value.

### PR DESCRIPTION
**Pull request by aiwantaozi**
Problem:
both alert group and alert rule advance option is confused, we need to
make then simple

Solution:
alert rule use alert group level advance option as default but user can
edit if they don't have to inherit from group

Issue:
https://github.com/rancher/rancher/issues/17538

### CODE REVIEW

**Comment by aiwantaozi** [comment]
need to work with UI

**Comment by orangedeng** [comment]
LGTM

File: apis/management.cattle.io/v3/alerting_types.go [hunk 1]
```diff
@@ -195,6 +195,7 @@ type CommonGroupField struct {
 type CommonRuleField struct {
 	DisplayName string `json:"displayName,omitempty"`
 	Severity    string `json:"severity,omitempty" norman:"required,options=info|critical|warning,default=critical"`
+	IsInherited *bool  `json:"isInherited,omitempty" norman:"default=true"`
```
Comment (cjellick) [inline apis/management.cattle.io/v3/alerting_types.go]: should just be `inherited` not `isInherited`

**Comment by loganhz** [comment]
Updated

**Comment by cjellick** [comment]
lgtm

**Pull request merged**